userguide/dailyuse/libertine.rst

```diff
@@ -56,6 +56,14 @@ Remove a package::
 .. note::
     If you have more than one container, then you can use the option ``-i CONTAINER-IDENTIFIER`` to specify for which container you want to perform an operation.
 
+Run desktop application
+-----------------------
+
+To run a graphical application you installed whose package installed a desktop file, i.e. ``gedit``, run in a terminal::
+
+  ubuntu-app-launch xenial_gedit_0.0
+
+If you are in doubt about what the argument must be, just start typing ``xenial`` and then find all the possible ones with the ``tab`` key.
```

Comment:
I think ``xenial`` here is the CONTAINERID, but I don't have a working device now to confirm. Can you check whether you get autocompletion also before you type any part of the appid? So, just ``ubuntu-app-launch[SPACE][TAB][TAB]". If so, please reformulate without "xenial", since other people might have different or multiple containers with other ids

Reply:
Check now https://github.com/ubports/docs.ubports.com/pull/248/commits/c520955cde9eb58b4532ae0f17e112f4b261b562.